src/org.hdfgroup.hdfview/hdf/view/TableView/DataProviderFactory.java

```diff
@@ -1641,7 +1643,10 @@ public Object getDataValue(int columnIndex, int rowIndex)
             super.getDataValue(columnIndex, rowIndex);
 
             try {
-                if (isUINT64)
+                if (isFLT16)
+                    //                  theValue = Float.toString(Float.float16ToFloat((short)theValue));
```

Comment:
Seems better to remove this and the below commented out line so we don't mislead ourselves into representing the values as Strings at some point in the future

Reply:
done